docs/getting-started/12-bind-redis-service-to-function.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Bind a Redis ServiceInstance to the Function
+type: Getting Started
+---
+
+This guide shows how you can bind a sample instance of the Redis service to the Function. After completing all the steps, you will get the Function with encoded Secrets to the service. You will use Redis as an external database for the Function. This database replaces the Function's in-memory storage.
+
+## Steps
+
+### Bind the Redis ServiceInstance to the Function
+
+Follow these steps:
+
+<div tabs name="steps" group="bind-redis-to-function">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Create a ServiceBinding custom resource (CR). Its **spec.instanceRef** field points to your Redis instance:
```

Comment:
Continuing [this](https://github.com/kyma-project/kyma/pull/9296/#discussion_r481983803) thread.

I believe this should stress more that you need to specify the Redis instance in this field.

If you don't want to go with my previous suggestion, 'd either leave your original version just unfolding the acronym: 
```suggestion
1. Create a ServiceBinding custom resource (CR) that points to the existing Redis instance in the **spec.instanceRef** field:
```

or change it to something like this: 
```
1. Create a ServiceBinding custom resource (CR) with its **spec.instanceRef** field pointing to your Redis instance:
```